### PR TITLE
Update sig testing subprojects

### PR DESCRIPTION
**Pull request by spiffxp**
Dropped gopherage and gubernator since they are inactive

Added kubetest2

### CODE REVIEW

File: sig-testing/README.md [hunk 1]
```diff
@@ -65,6 +57,10 @@ Kubernetes IN Docker. Run Kubernetes test clusters on your local machine using D
   - sigs.k8s.io/kind bi-weekly meeting: [Mondays at 11:00 PT (Pacific Time)](https://zoom.us/j/960461819) (bi-weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=11:00&tz=PT%20%28Pacific%20Time%29).
     - [Meeting notes and Agenda](https://docs.google.com/document/d/1b9Ppm7ZT_tMWRs5Ph1zGJJKb5nF9c3ZHzMwg1olJIrc/edit).
     - [Meeting recordings](https://bit.ly/k8s-sig-testing-videos).
+### kubetest2
+Kubetest2 is intended to be the next significant iteration of kubetest
```
Comment (BenTheElder) [inline sig-testing/README.md:61]: we should include kubetest, but note that we're deprecating it.

File: sig-testing/README.md [hunk 2]
```diff
@@ -65,6 +57,10 @@ Kubernetes IN Docker. Run Kubernetes test clusters on your local machine using D
   - sigs.k8s.io/kind bi-weekly meeting: [Mondays at 11:00 PT (Pacific Time)](https://zoom.us/j/960461819) (bi-weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=11:00&tz=PT%20%28Pacific%20Time%29).
     - [Meeting notes and Agenda](https://docs.google.com/document/d/1b9Ppm7ZT_tMWRs5Ph1zGJJKb5nF9c3ZHzMwg1olJIrc/edit).
     - [Meeting recordings](https://bit.ly/k8s-sig-testing-videos).
+### kubetest2
```
Comment (BenTheElder) [inline sig-testing/README.md:60]: pretty sure you mispelled "kates" cc @amwat

Comment (amwat) [reply sig-testing/README.md:60]: leave a vote on the doc 😬 
shouldn't block this PR, will rename it here if we finalize on the rename.
Will definitely finalize the name before requesting for a new repo though 👍 

Thanks!

/lgtm
/hold

File: sig-testing/README.md [hunk 3]
```diff
@@ -47,14 +47,6 @@ Boskos is a resource manager service that handles different kinds of resources a
 - **Owners:**
   - https://raw.githubusercontent.com/kubernetes-sigs/boskos/master/OWNERS
   - https://raw.githubusercontent.com/kubernetes/test-infra/master/boskos/OWNERS
-### gopherage
-Gopherage is a tool for manipulating Go coverage files. We use it on the Kubernetes project to report on code coverage due to e2e tests
```
Comment (amwat) [inline sig-testing/README.md:51]: Are we sure we want to drop this?
It is unmaintained but is currently ["used" in CI](https://github.com/kubernetes/test-infra/blob/c68834a35762399bac39eff4bbcef65cb5b9fd12/config/jobs/kubernetes/sig-testing/coverage.yaml#L46) without any planned replacement yet. 
should we wait for a replacement or until is completely unused?
FYI @MushuEE

Comment (spiffxp) [reply sig-testing/README.md:51]: I feel like this is still covered under the test-infra subproject.  There are a variety of codebases in there that are used but in maintenance mode.  If this reaches a level of activity where we're making plans for it, have a team of people rallying around it, or are thinking of extracting it from test-infra, then I think it makes sense to be its own subproject.

File: sig-testing/README.md (hunk 1 above)
Comment (spiffxp) [reply sig-testing/README.md:61]: So I'm tempted to either:
- lump that in with kubetest2 and call it the kubetest subproject
- make a 'legacy-test-infra' subproject, and bin things like bootstrap and kubetest under it

Comment (spiffxp) [reply sig-testing/README.md]: I opted for @BenTheElder's suggestion, leaving this named as kubetest2 but noting kubetest will be deprecated

File: sig-testing/README.md (hunk 3 above)
Comment (BenTheElder) [reply sig-testing/README.md:51]: sgtm

**Pull request merged**